webrtc.html

```diff
@@ -2703,9 +2703,9 @@
                         </ol>
                       </li>
                       <li>
-                        <p>If <code><var>candidate</var>.ufrag</code> is neither
+                        <p>If <code><var>candidate</var>.usernameFragment</code> is neither
                         <code>undefined</code> nor <code>null</code>, and is not
-                        equal to any ufrag present in the corresponding
+                        equal to any usernameFragment present in the corresponding
```

Comment:
Since this isn't code, should just say "username fragment" here like ICE itself does.